plugins/modules/restore.py

```diff
@@ -138,7 +138,7 @@
   replace_db_name_in_file:
     description:
       - If switch set any occurrence of the original database's name in a data or log file
-        will be replace with the name specified in the DatabaseName parameter.
+        will be replace with the name specified in the I(database_name) option.
```

Comment:
This comment is actually for line 147, I missed the `ALL` -> `B(all)` recommendation for that one (I just happened to browse the rendered docs and noticed it there): https://lowlydba.github.io/lowlydba.sqlserver/pr/119/collections/lowlydba/sqlserver/restore_module.html#parameter-destination_file_prefix

Reply:
I recommend giving all the rendered docs a once over too, to see if you find anything I didn't spot, since we have this open.